README.md

```diff
@@ -39,17 +39,17 @@ Each set of unique sequences or API endpoint tests should be started with new `f
 
 ```javascript
 
-var frisby = require('icedfrisby');  // get IcedFrisby with `npm install icedfrisby`
-var Joi = require('joi'); // get Joi with `npm install joi`
+const frisby = require('icedfrisby')  // Get IcedFrisby with `npm install icedfrisby`.
+const Joi = require('joi') // Joi is installed along with IcedFrisby.
```

Comment:
Hmm, I haven't had that problem… I'd agree it's probably not a good idea to require it unless it's declared separately, though.

I'll revert this comment change. Let's discuss further how to handle this.